<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>699</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(225, 255, 247);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>862</width>
          <height>628</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QFrame" name="frame_2">
             <property name="minimumSize">
              <size>
               <width>139</width>
               <height>608</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>608</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 127);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QWidget" name="horizontalLayoutWidget_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>151</width>
                <height>611</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="1">
                <widget class="QFrame" name="frame_3">
                 <property name="maximumSize">
                  <size>
                   <width>169</width>
                   <height>619</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 170, 0);</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <widget class="QWidget" name="verticalLayoutWidget">
                  <property name="geometry">
                   <rect>
                    <x>20</x>
                    <y>130</y>
                    <width>101</width>
                    <height>331</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QPushButton" name="pushButton">
                     <property name="text">
                      <string>Page 1</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_2">
                     <property name="text">
                      <string>Page 2</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_3">
                     <property name="text">
                      <string>Page 3</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QStackedWidget" name="stackedWidget">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>0</y>
                <width>631</width>
                <height>611</height>
               </rect>
              </property>
              <property name="currentIndex">
               <number>2</number>
              </property>
              <widget class="QWidget" name="page">
               <widget class="QFrame" name="frame_4">
                <property name="geometry">
                 <rect>
                  <x>160</x>
                  <y>200</y>
                  <width>120</width>
                  <height>80</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(170, 255, 127);</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </widget>
              <widget class="QWidget" name="page_3">
               <widget class="QFrame" name="frame_5">
                <property name="geometry">
                 <rect>
                  <x>190</x>
                  <y>230</y>
                  <width>120</width>
                  <height>80</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(0, 170, 127);</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </widget>
              <widget class="QWidget" name="page_2">
               <widget class="QColumnView" name="columnView">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>50</y>
                  <width>601</width>
                  <height>531</height>
                 </rect>
                </property>
               </widget>
               <widget class="QScrollArea" name="scrollArea">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>50</y>
                  <width>601</width>
                  <height>531</height>
                 </rect>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>599</width>
                   <height>529</height>
                  </rect>
                 </property>
                 <widget class="QFrame" name="frame_6">
                  <property name="geometry">
                   <rect>
                    <x>190</x>
                    <y>130</y>
                    <width>120</width>
                    <height>80</height>
                   </rect>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(170, 85, 127);</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </widget>
               </widget>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
